tests/scenarios/compat-resolver-test.ts

```diff
@@ -525,7 +525,7 @@ Scenarios.fromProject(() => new Project())
           }
         );
 
-        expectTranspiled('templates/application.hbs').equalsCode(`          
```

Comment:
oof, lint configs that don't trim trailing invisible characters make for messy PRs :( 
Gonna try to PR a fix for this across the board

Reply:
oh! apparently this isn't a lint thing, and it's an intentional choice of prettier to not trim trailing spaces within template s trings. :thinking: 
Guess I'll tweak my editor, and re-commit

Reply:
no! it's the editorconfig that does this behavior! so we need an editorconfig lint :thinking:

Reply:
Whitespace inside strings is semantically important. If your editor is trimming it, it's wrong.

Reply:
editorconfig is trimming it, so its wrong :shrug: 

why have editorconfig at all, I guess?

Reply:
These changes are no longer a part of this PR, btw